spec.emu

```diff
@@ -248,7 +250,12 @@ contributors: Google, Ecma International
         1. If the result of ! IsValidCodeForDisplayNames(_displayNames_.[[Type]], _code_) is *false*, throw a *RangeError* exception.
         1. Let _fields_ be _displayNames_.[[Fields]].
         1. Let _name_ be _fields_[[_code_]].
-        1. Return _name_.
+        1. If _name_ is not *undefined*, then
+          1. Return _name_.
+        1. Let _fallback_ be _displayNames_.[[Fallback]].
```

Comment:
seems like the `_fallback_` var isn't really needed, could do `If _displayNames_.[[Fallback]] is`?